_includes/footer.md

```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```

Comment:
> I added the trademark to the main page using a front matter variable in 5d11346. Does this work?

This looks great to me. I appreciate you being flexible.

> Is this even required?

tl;dr Given my current understanding, yes.

I think this is something that @billierinaldi and I had recently added to the website specifically to meet the branding requirements that the ASF sets forward. See http://www.apache.org/foundation/marks/pmcs.html#attributions:

> All project or product homepages must feature a prominent trademark attribution of all applicable Apache trademarks. Other project pages should feature attributions of any marks displayed thereon.

Policy aside, I believe reminding visitors of the website that all of these names (*full* names, even) are trademarks of the ASF and deserve proper attribution is a worthwhile goal. It really doesn't strike me as intrusive or distracting.